src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -32,8 +32,17 @@
     void testMapPathologyRecordToBundle() {
```

Comment:
Can you add a test mapping a pathology record to bundle where there is no performer (Practitioner)? Covering your `ifPresent` logic in the `BundleMapper`

Reply:
I've tried, but I'm not 100% how to test this scenario efficiently, because the size of the variable will change as we add more mappers so this test will have to be changed constantly. 

The result at the moment is that there's one less resource in the Bundle so I've tested that and that the types of the two resources in the Bundle are `Practitioner` and `Patient` respectively.

Reply:
I think what you have done looks good - thank you 👍